hudi-sync/hudi-hive-sync/src/main/java/org/apache/hudi/hive/HiveSyncTool.java

```diff
@@ -117,7 +118,9 @@ private void initTableNameVars(HiveSyncConfig config) {
           this.roTableName = Option.empty();
           break;
         case MERGE_ON_READ:
-          this.snapshotTableName = tableName + SUFFIX_SNAPSHOT_TABLE;
+          this.snapshotTableName = config.getBoolean(HIVE_SKIP_RT_SUFFIX_FOR_READ_SNAPSHOT_TABLE)
+                  ? tableName
+                  : tableName + SUFFIX_SNAPSHOT_TABLE;
```

Comment:
What's the situation if both `skip_ro_suffix` and `skip_rt_suffix` are true.

Reply:
Thanks for your reply.That's very thoughtful of you. Can we throw an exception in this case? Any suggestion do you have.

Reply:
We can throw exception if both options are valuable, but before do that, let's figure out the background why `HIVE_SKIP_RO_SUFFIX_FOR_READ_OPTIMIZED_TABLE` is introduced ?

Reply:
During Hive query, first, we prefer to use the original table name, and second, we query the latest data.